DDS.mwc

```diff
@@ -12,6 +12,7 @@ workspace {
     ACE_wrappers
     // For the case where the user clones git repository
     ATCD
+    ACE_TAO
```

Comment:
At the moment an user has still a local git clone from before the rename he could still use ATCD as name, that is why I didn't remove the ATCD line